pkg/controller/vpc_nat_gw_eip.go

```diff
@@ -249,11 +248,11 @@ func (c *Controller) handleAddIptablesEip(key string) error {
 	if eipV4Cidr, err = c.getEipV4Cidr(v4ip); err != nil {
 		return err
 	}
-	if gw, err = c.GetGw(); err != nil {
+	if v4Gw, _, err = c.GetGwbySubnet(util.VpcExternalNet); err != nil {
```

Comment:
Also log error here to fix the legacy style issues

Reply:
done